node/sc/bridge_account_manager.go

```diff
@@ -39,34 +43,88 @@ type accountInfo struct {
 // BridgeAccountManager manages bridge account for main/service chain.
 type BridgeAccountManager struct {
 	// TODO-Klaytn need to consider multiple bridge accounts?
-	mcAccount *accountInfo
-	scAccount *accountInfo
+	pAccount *accountInfo
+	cAccount *accountInfo
 }
 
 // NewBridgeAccountManager returns bridgeAccountManager created by main/service bridge account keys.
-func NewBridgeAccountManager(mcKey, scKey *ecdsa.PrivateKey) (*BridgeAccountManager, error) {
+func NewBridgeAccountManager(dataDir string) (*BridgeAccountManager, error) {
+	pWallet, pAcc, isLock, err := InitializeBridgeAccountKeystore(path.Join(dataDir, "parent_bridge_account"))
+	if err != nil {
+		return nil, err
+	}
+
+	if isLock {
+		logger.Warn("parent_bridge_account is locked. Please unLock manually for Service chain")
+	}
+
+	cWallet, cAcc, isLock, err := InitializeBridgeAccountKeystore(path.Join(dataDir, "child_bridge_account"))
+	if err != nil {
+		return nil, err
+	}
+
+	if isLock {
+		logger.Warn("child_bridge_account is locked. Please unLock manually for Service chain")
+	}
+
+	logger.Info("bridge account is loaded", "parent", pAcc.String(), "child", cAcc.String())
+
 	mcAcc := &accountInfo{
-		key:      mcKey,
-		address:  crypto.PubkeyToAddress(mcKey.PublicKey),
+		wallet:   pWallet,
+		address:  pAcc,
 		nonce:    0,
 		chainID:  nil,
 		gasPrice: nil,
 	}
 
 	scAcc := &accountInfo{
-		key:      scKey,
-		address:  crypto.PubkeyToAddress(scKey.PublicKey),
+		wallet:   cWallet,
+		address:  cAcc,
 		nonce:    0,
 		chainID:  nil,
 		gasPrice: nil,
 	}
 
 	return &BridgeAccountManager{
-		mcAccount: mcAcc,
-		scAccount: scAcc,
+		pAccount: mcAcc,
+		cAccount: scAcc,
 	}, nil
 }
 
+// ImportUnlockAccountKeystore initialize a keystore, import exist keys, and try to unlock the key for bridge account.
+func InitializeBridgeAccountKeystore(keystorePath string) (accounts.Wallet, common.Address, bool, error) {
+	ks := keystore.NewKeyStore(keystorePath, keystore.StandardScryptN, keystore.StandardScryptP)
+
+	if len(ks.Accounts()) == 0 {
+		pwdStr := setup.RandStringRunes(100)
+		acc, err := ks.NewAccount(pwdStr)
+		if err != nil {
+			return nil, common.Address{}, true, err
+		}
+		setup.WriteFile([]byte(pwdStr), keystorePath, acc.Address.String())
+
+		if err := ks.Unlock(acc, pwdStr); err != nil {
+			logger.Warn("bridge account wallet unlock is failed by created password file.", "address", acc.Address, "err", err)
+			return ks.Wallets()[0], acc.Address, true, nil
+		}
+
+		return ks.Wallets()[0], acc.Address, false, nil
+	}
+
+	acc := ks.Accounts()[0]
+	pwdFilePath := path.Join(keystorePath, acc.Address.String())
+	pwdStr, err := ioutil.ReadFile(pwdFilePath)
+	if err == nil {
```

Comment:
If `ioutil.ReadFile` failed, the code does not show any error or warning. 
I think it would be better if the code returns an error for the error case.
```
if err != nil {
    return nil, common.Address{}, true, err
}
if err := ks.Unlock(acc, string(pwdStr)); err != nil {
...
```

Reply:
Thanks for leaving the comment. :)
I think logging warning is better.
This reading file is optional feature. If it is not possible to unlock with password file, it is ok.
In this code, 3rd return value represents if the account is unlocked or not. 
In this function, It doesn't know this account is parent or child account. So through 3rd return value, after return , base method will printed warn log. In that case, user should unlock the account via API.

Reply:
I understand your intention. Thank you for the response.